instrumentation/src/main/java/io/opentelemetry/android/config/OtelRumConfig.java

```diff
@@ -119,4 +125,15 @@ public void disableNetworkChangeMonitoring() {
     public boolean isNetworkChangeMonitoringEnabled() {
         return this.networkChangeMonitoringEnabled;
     }
+
+    /** Adds a new initialization listener. It will be called during the RUM initialization. */
+    public void addInitializationListener(InitializationListener listener) {
+        if (!initializationListeners.contains(listener)) {
```

Comment:
It could be a `Set` if the goal is to not allow dupes anyway.

Reply:
Good point, I've added the changes 👍